programs/merkle-distributor/src/instructions/admin/new_distributor.rs

```diff
@@ -127,6 +132,19 @@ pub struct NewDistributor<'info> {
     )]
     pub distributor: AccountLoader<'info, MerkleDistributor>,
 
+    /// [MerkleDistributor].
+    #[account(
+        init,
+        seeds = [
+            b"PartialMerkleTree".as_ref(),
+            distributor.key().to_bytes().as_ref(),
+        ],
+        bump,
+        space = PartialMerkleTree::space(total_node as usize),
+        payer = admin
+    )]
+    pub partial_merkle_tree: Account<'info, PartialMerkleTree>,
```

Comment:
save nodes in distributor state

Reply:
Because AccountLoader<T> does not support storing a dynamic Vec<T> directly. So, create `CanopyTree` to storage metadata (root, nodes, depth..) of MerkleDistributor